emhttp/plugins/dynamix/styles/default-white.css

```
@@ -60,8 +60,8 @@ div.title span.right{font-size:1.4rem;padding-top:2px;padding-right:10px;float:r
 div.title span img{padding-right:4px}
 div.title.shift{margin-top:-30px}
 #menu{position:absolute;top:90px;left:0;right:0;display:grid;grid-template-columns:auto max-content;z-index:101}
-.nav-tile{height:4rem;line-height:4rem;padding:0;margin:0;font-size:1.2rem;letter-spacing:1.8px;background-color:#1c1b1b;white-space:nowrap;overflow-x:auto;overflow-y:hidden;scrollbar-width:thin}
-.nav-tile::-webkit-scrollbar{height:5px}
+.nav-tile{height:4rem;line-height:4rem;padding:0;margin:0;font-size:1.2rem;letter-spacing:1.8px;background-color:#1c1b1b;white-space:nowrap;overflow-x:auto;overflow-y:hidden;scrollbar-height:4px}
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)